.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,27 @@
 ## Proposed Commit Message
 <!-- Include a proposed commit message because all PRs can be merged in a variety of ways by the reviewer -->
 
-> summary: no more than 70 characters
->
-> A description of what the change being made is and why it is being
-> made, if the summary line is insufficient.  The blank line above is
-> required. This should be wrapped at 72 characters, but otherwise has
-> no particular length requirements.
->
-> If you need to write multiple paragraphs, feel free.
->
-> LP: #NNNNNNN (replace with the appropriate Launchpad bug reference if applicable)
-> Fixes: #NNNNNNN (replace with the appropriate github issue if applicable)
+```
+summary: no more than 70 characters
+
+A description of what the change being made is and why it is being
+made, if the summary line is insufficient.  The blank line above is
+required. This should be wrapped at 72 characters, but otherwise has
+no particular length requirements.
+
+If you need to write multiple paragraphs, feel free.
+
+LP: #NNNNNNN (replace with the appropriate Launchpad bug reference if applicable)
+Fixes: #NNNNNNN (replace with the appropriate github issue if applicable)
+```
 
 ## Test Steps
 <!-- Please include any steps necessary to verify (and reproduce if
 this is a bug fix) this change on a live deployed system,
 including any necessary configuration files, user-data,
 setup, and teardown. Scripts used may be attached directly to this PR. -->
 
-## Desired commit type::
+## Desired commit type:
```

Comment:
I am strongly against removing the double colon, it is a valuable visual enhancement to the title :-1: 


Kidding - but (for OCD reasons) we could have no colon at all (like in the previous two titles) or one colon for each title.

Reply:
I would honestly prefer a [`::<>`](https://turbo.fish) but thought this might be more widely accepted :shrug: 

I left it so that the two sections with checklists have colons and the two sections with written content don't have colons. I can totally change it though. I would lean towards no colons on any titles I think actually

Reply:
Ok brains registered your suggestion as a standard: no strong opinion

Reply:
Ha yes :) Just updated to remove all colons